mappings/net/minecraft/world/chunk/Chunk.mapping

```diff
@@ -57,18 +67,24 @@ CLASS none/atp net/minecraft/world/chunk/Chunk
 	METHOD a addBlockEntity (Lnone/cn;Lnone/are;)V
 		ARG 0 pos
 		ARG 1 blockEntity
-	METHOD a (Lnone/cn;Lnone/asm;)Lnone/asm;
+	METHOD a setBlockState (Lnone/cn;Lnone/asm;)Lnone/asm;
 		ARG 0 pos
 		ARG 1 state
-	METHOD a (Lnone/cn;Lnone/atp$a;)Lnone/are;
+	METHOD a getBlockEntity (Lnone/cn;Lnone/atp$a;)Lnone/are;
 		ARG 0 pos
+		ARG 1 mode
 	METHOD a (Lnone/es;IZ)V
 		ARG 0 buffer
 	METHOD a addEntity (Lnone/sg;)V
 		ARG 0 entity
 	METHOD a removeEntity (Lnone/sg;I)V
 		ARG 0 entity
 		ARG 1 chunkY
+	METHOD a getEntitiesAndPartsInAreaExcludingOne (Lnone/sg;Lnone/bcs;Ljava/util/List;Lcom/google/common/base/Predicate;)V
```

Comment:
This seems like a very long name. Would something like `getEntitiesInAreaExcluding` be an accurate name for it?